SUSE/rmt
# Yabeda integration for prom metrics

## Description



* Related Issue / Ticket / Trello card: https://trello.com/c/Mus496Nz/3617-repct-integrate-yabeda-for-prometheus-collectors

## How to test 

Start the rails server:
`> bin/rails s`
On a separate console, run:
`> curl localhost:4224/metrics`

Should see a text with metrics and values

## Change Type

*Please select the correct option.*


## Checklist

*Please check off each item if the requirement is met.*


## Review

Please check out our [review guidelines](https://github.com/SUSE/scc-docs/blob/master/team/workflow/code_review.md) 
and get in touch with the author to get a shared understanding of the change.
